frontend/src/components/chats/components/current-chat/components/messages-list/messages-list.tsx

```diff
@@ -0,0 +1,39 @@
+import defaultAvatar from 'assets/img/avatar-default.svg';
+import { ChatMessageGetAllItemResponseDto, FC } from 'common/types/types';
+import { getFormattedDate } from 'helpers/helpers';
+
+import { Message } from './components/components';
+import styles from './styles.module.scss';
+
+type Props = {
+  currentUserId: number;
+  messages: ChatMessageGetAllItemResponseDto[];
+};
+
+const MessagesList: FC<Props> = ({ messages, currentUserId }) => {
+  return (
+    <div className={styles.messagesListWrapper}>
+      {messages.map((message) => {
+        return (
+          <div key={message.id} className={styles.messageWrapper}>
+            <Message
+              messageAuthor={
+                message.sender.id === currentUserId ? 'user' : 'opponent'
+              }
+              content={message.message}
+              postTime={
+                getFormattedDate(message.createdAt, 'HH:mm, dd.MM').split(
```

Comment:
Have you tried `getFormattedDate(new Date(message.createdAt), 'HH:mm')`?

Reply:
@andanfe `getFormattedDate` accepts a string and runs `new Date(...)` inside that helper, so I guess that shouldn't be an issue here

Reply:
Agreed

Reply:
Oh, I missed that 🤔 Just tried on playground date-fns with `HH:mm` and it works for `createdAt` date-string format :c

Reply:
you can use another format date. avoid additional logic. you should use only `getFormattedDate `

Reply:
👍 for another format, why not just `HH:mm`? 
+ we can avoid .split(',')[0]

Reply:
Yesterday `HH:mm` format did not work, returned undefined, today it is ok. I have no idea how could that happen